compiler/circle-mpqsolver/src/bisection/Quantizer.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __MPQSOLVER_BISECTION_QUANTIZER_H__
+#define __MPQSOLVER_BISECTION_QUANTIZER_H__
+
+#include <luci/IR/Module.h>
+#include <luci/CircleQuantizer.h>
+
+#include <string>
+#include <vector>
+#include <memory>
+
+namespace mpqsolver
+{
+namespace bisection
+{
+
+using LayerParam = luci::CircleQuantizer::Options::LayerParam;
+using LayerParams = std::vector<std::shared_ptr<LayerParam>>;
+
+class Quantizer final
+{
+public:
+  /**
+   * @brief quantize recorded module (min/max initialized) with specified parameters
+   * returns true on success
+   */
+  static bool quantize(luci::Module *module, const std::string &quant_dtype,
```

Comment:
(optional) these two methods are wrapped in `class Quantizer` with static methods.
can these be moved outside of `Quantizer`?
just like
```cxx
bool quantize(luci::Module *module, const std::string &quant_dtype, ...);
bool fake_quantize(luci::Module *module, const std::string &quant_dtype, ...);
```

Reply:
if there are other purpose than `Quantize` in `bisection` then putting these two methods inside `class Quantize` is OK.

Reply:
ok

Reply:
> if there are other purpose than `Quantize` in `bisection` then putting these two methods inside `class Quantize` is OK.

Currently it's just quantize and fake_quantize. I'll remove them.